docs/_drafts/scv2-operator.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Seldon Core v2 Deep Dive---Operator
+tags: seldon seldon-core-v2 architecture kubernetes
+---
+
+This article is part of a series about [Seldon Core v2](https://github.com/SeldonIO/seldon-core/tree/v2).
+
+Seldon Core v2 is largely platform agnostic by design, with the **operator** existing to help integrate it natively into Kubernetes.
+It's responsible for managing the Kubernetes resources related to Core v2, and acts as an adapter between those resources and Core v2's internal representation.
+
+This article discusses the operator's role in more detail, including what those Kubernetes resources are, how it handles them, and how it was built.
+Of particular interest is the operator's use of **meta-resources**, i.e. resources that define other resources.
+
+You might be interested in this article if:
+* You're interested in Seldon Core v2 and understanding its components in depth;
+* You're considering writing an operator to integrate your own (open-source) project into Kubernetes; or
+* You're interested in how Kubernetes resources and operators work and how to reason about them.
+
+## Core v1 & Kubernetes --- a match made in heaven?
```

Comment:
Although I'll keep writing it for now, I suspect a lot of this content would fit better into a post on SCv1 vs. SCv2.  This post is already starting to look a bit long, and a lot of that content is only tangentially related to the operator.